CHANGELOG.md

```diff
@@ -1,9 +1,35 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+This project adheres to [Semantic Versioning](http://semver.org/)
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## 1.2.0 2017-05-13
+## [Unreleased]
+
+## [1.3.0] - 2018-02-04
+### Added
+- Initial support for OpenBSD! (@smbambling)
+- Ubuntu now get's `apt-transport-https` installed to support HTTPS repos. (@kevit)
+- Default to HTTPS APT repo's.  @jaredledvina
+- Ubuntu - Redis now always binds to `0.0.0.0` to ensure it's accessible. (@tculp)
+- Allow for configuring when a node gets the `sensu-client` config file. (@tculp)
+- Split up the variables used to determine if a host gets rabbitmq/redis for more flexibility in deployments. (@tculp)
+- Allow for deploying client definitions based on groups. (@tculp)
+- Default to HTTPS Yum repo's and install the Yum key for package signing validation via HTTPS.  (@jaredledvina)
+- Used HTTPS for APT key.  (@jaredledvina)
+- Amazon Linux now get's proper yum repo and supports Amazon Linux 2. (@romainrbr)
+- Yum based distros now get EPEL to support installing a newer and supported version of RabbitMQ. (@romainrbr)
+- CentOS now supports using Bintray mirrors for installing RabbitMQ to work around Erlang issues with older versions. (@romainrbr)
+- All PR's are now required to pass TravisCI tests.  (@jaredledvina)
+- Ensure that we configure the `mode` and `umask` for files to work in a more restrictive environment. (@roumano)
```

Comment:
:100: thanks for working through the submitter's issue, I am sure they greatly appreciate it.